README.md

```diff
@@ -1,2 +1,3 @@
 # fiware-health
 Program to check the functionality of each FIWARE Lab node.
```

Comment:
You should add here more description about the execution of this component, license and so on...